rules/windows/defense_evasion_suspicious_execution_from_mounted_device.toml

```
@@ -0,0 +1,81 @@
+[metadata]
+creation_date = "2021/05/28"
+maturity = "production"
+updated_date = "2021/05/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when a script interpreter or signed binary is launched via a non-standard working directory. An attacker may
+use this technique to evade defenses.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Execution from a Mounted Device"
+references = [
+    "https://www.microsoft.com/security/blog/2021/05/27/new-sophisticated-email-based-attack-from-nobelium/",
+    "https://www.volexity.com/blog/2021/05/27/suspected-apt29-operation-launches-election-fraud-themed-phishing-campaigns/",
+]
+risk_score = 47
+rule_id = "8a1d4831-3ce6-4859-9891-28931fa6101d"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type : "start" and process.executable : "C:\\*" and
+  process.working_directory : "?:\\" and not process.working_directory: "C:\\" and
```

nit: this is not needed for precedence, but I think it helps with readability of the intent of the logic

```suggestion
  (process.working_directory : "?:\\" and not process.working_directory: "C:\\") and
```